easybuild/easyconfigs/p/Perl/Perl-5.26.0-foss-2017b.eb

```diff
@@ -1149,6 +1149,16 @@ exts_list = [
         'source_urls': ['http://search.cpan.org/CPAN/authors/id/D/DE/DEWEG/'],
         'checksums': ['c3d28ad479b14b803bea3ccb8e90cb4c6a2565ee70866a0e9cfe4d76d4d45352'],
     }),
+    ('Math::CDF', '0.1', {
+        'source_tmpl': 'Math-CDF-%(version)s.tar.gz',
+        'source_urls': ['http://search.cpan.org/CPAN/authors/id/C/CA/CALLAHAN/'],
+        'checksums': ['7896bf250835ce47dcc813cb8cf9dc576c5455de42e822dcd7d8d3fef2125565'],
+    }),
 ]
 
+sanity_check_paths = {
+    'files': ['bin/perl', 'bin/perldoc'],
+    'dirs': ['lib/5.26.0', 'man']
```

Comment:
I don't understand why this needs to be changed, how can this suddenly be different?

In any case, this must be tackled in the custom `Perl` easyblock we have for `Perl`, not here.

Reply:
I don't understand either, that's why I submitted this hoping you would :)
Of course I agree that it should be fixed in the easyblock. This is just to show what should be changed.

Reply:
@smoors Do you have more details? Which OS? Can you tell when this started occurring? After an OS update or something?

Reply:
Hm. When I install it on our production machines it uses lib/perl5/5.26.0, so this is something specific to my environment. I have 'perl' in my install path, maybe there is a subtle bug in the parsing of the path or something.
You may ignore this change.